build_config.sample.yaml

```diff
@@ -21,5 +21,5 @@ modules:
     # An example of pulling a module from NPM
     - "@vector-im/element-web-ilag-module@^0.0.1"
 
-    # An example of pulling a module from github
-    - "github:vector-im/element-web-ilag-module#main"
+    # An example of pulling a module from local filesystem during development
+    - "file:/home/user/development/element-web-ilag-module"
```

Comment:
Do we need to mention here that you'll have to build it yourself in the folder?

Reply:
That depends on your module though, if your module is JS then you don't need to build, nowhere do we state that we build your module, when installing an npm dependency it may or may not be built depending on what the lifecycle scripts do. So this could cause more confusion than good IMO

Reply:
Ok, I see. 👍